Makefile.in

```diff
@@ -80,7 +89,7 @@ EXTRA_PCRE2 = @EXTRA_PCRE2@
 # Various flags
 #
 CXXFLAGS = @CXXFLAGS@ $(EXTRA_CXXFLAGS)
-CPPFLAGS = -DLOCALEDIR=\"$(localedir)\" -DPREFIX=L\"$(prefix)\" -DDATADIR=L\"$(datadir)\" -DSYSCONFDIR=L\"$(sysconfdir)\" -DBINDIR=L\"$(bindir)\" -DDOCDIR=L\"$(docdir)\" -iquote. -iquote./src/
+CPPFLAGS = -D_UNICODE=1 -DLOCALEDIR=\"$(localedir)\" -DPREFIX=L\"$(prefix)\" -DDATADIR=L\"$(datadir)\" -DSYSCONFDIR=L\"$(sysconfdir)\" -DBINDIR=L\"$(bindir)\" -DDOCDIR=L\"$(docdir)\" -iquote. -iquote./src/
```

Comment:
"I've got a bad feeling about this" - I suspect `-D_UNICODE` may have side effects on Windows, although I think it's only under Visual Studio - @andrex-e-schulman may be able to comment.